components/popover/index.css

```diff
@@ -111,9 +111,15 @@ governing permissions and limitations under the License.
 .spectrum-Popover--top-right,
 .spectrum-Popover--top-start,
 .spectrum-Popover--top-end {
-  margin-bottom: var(--mod-popover-offset, var(--spectrum-popover-offset));
+  /* spacing to include tip in calculation of offset from source */
+  &.spectrum-Popover--withTip {
+    /* tip size minus where it overlaps with popover border */
+    margin-bottom: calc(var(--mod-popover-pointer-height, var(--spectrum-popover-pointer-height)) - var(--mod-popover-border-width, var(--spectrum-popover-border-width)));
+  }
+
   /* popover animates upward ⬆ */
   &.is-open {
+    --spectrum-overlay-animation-distance: var(--mod-popover-animation-distance, var(--spectrum-popover-animation-distance));
```

Comment:
Won't this immediately get overwritten by the same variable that's defined in the `%spectrum-overlay--top--open`? Also, I'm wondering if it makes more sense to just use the one `transform` line it's importing with new values rather than needing the `@inherit` and probably needing to move the custom property after the `@inherit`.

Initially I was thinking that this variable could be defined once in `.spectrum-Popover.is-open` since it appears to be defined the same way for is-open on all (?) the variants.

Reply:
Yeah, per [that convo](https://github.com/adobe/spectrum-css/pull/2266#discussion_r1383879047) in the tooltip PR (#2266), I added a commit that reworks this to define the transforms in `popover/index.css` instead 👍